ddouglasz/marley-spoon-frontend
# Feat/data fetching

**What does this PR do?**

 - This PR connect app to API and fetch all needed data for the frontend.

 **Description of Task to be completed?**

 - User can see a list of recipes on the index page
 - User can click on a recipe and navigate to the recipe details page


 **How should this be manually tested?**

 - Get the app up and running
 - Check the browser to see basic display of recipe contents

 **Any background context you want to provide?**

 xxx

 **What are the relevant stories?**

 xxx

 **Screenshots (if appropriate)**

 xxx

 **Questions:**

xxx
